app/components/custom/budgets/phases_component.html.erb

```diff
@@ -0,0 +1,67 @@
+<section class="budget-phases">
+  <header>
+    <h2><%= t("budgets.index.all_phases") %></h2>
+  </header>
+
+  <ul class="phases-list tabs" data-tabs id="budget_phases_tabs" data-deep-link="true">
+    <% phases.each do |phase| %>
+      <li class="phase-title tabs-title <%= "is-active current-phase-tab" if phase.current? %>">
+        <a href="#<%= phase_dom_id(phase) %>">
+          <% if phase.current? %>
+            <span class="current-phase-timeline"><%= t("budgets.index.current_phase") %></span>
+          <% end %>
+
+          <span class="phase-name"><%= phase.name %></span>
+        </a>
+      </li>
+    <% end %>
+  </ul>
+
+  <div class="tabs-content" data-tabs-content="budget_phases_tabs">
+    <% phases.each do |phase| %>
+      <div id="<%= phase_dom_id(phase) %>" class="tabs-panel <%= "is-active" if phase.current? %>">
+        <!--
```

Comment:
This comment and the closing tag is the only addition compared to the original file.